test/serializers/development_serializer_test.rb

```diff
@@ -65,7 +65,7 @@ def serializer_except
   private
 
     def expected_row
-      [980190962, 562391268, "1970-01-01 00:00:00 UTC", "1970-01-01 00:00:00 UTC", nil, nil, nil, nil, nil, nil, "Godfrey Hotel", "in_construction", nil, nil, nil, nil, "505 Washington Street", "Boston", "MA", "02111", nil, nil, 2016, nil, nil, nil, nil, nil, nil, nil, nil, 75, 12, nil, nil, nil, nil, false, false, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, "Metropolitan Area Planning Council", "http://mapc.org", nil, "Boston, MA", nil, "MAPC", "MAPC", "landlord"]
+      [980190962, 562391268, "1970-01-01 00:00:00 UTC", "1970-01-01 00:00:00 UTC", nil, nil, nil, nil, nil, nil, "Godfrey Hotel", "in_construction", nil, nil, nil, nil, "505 Washington Street", "Boston", "MA", "02111", nil, nil, 2016, nil, nil, nil, nil, nil, nil, nil, nil, 75, 12, nil, nil, nil, nil, false, true, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, "Metropolitan Area Planning Council", "http://mapc.org", nil, "Boston, MA", nil, "MAPC", "MAPC", "landlord"]
```

Comment:
Line is too long. [484/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).